packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -74,9 +96,9 @@ export class GuxSelectorCard {
     validateFormIds(this.root, this.input);
   }
 
-  private setChecked(): void {
-    this.input.checked = true;
-    this.checked = true;
+  private setChecked(newValue: boolean): void {
+    this.input.checked = newValue;
+    this.checked = newValue;
```

Comment:
It might be better to see if we can listen to the checked event from the input to keep these in sync.

Reply:
I scrapped this stuff and used CSS to show the border on the input instead